django/db/backends/sqlite3/creation.py

```diff
@@ -85,6 +98,9 @@ def _clone_test_db(self, suffix, verbosity, keepdb=False):
             except Exception as e:
                 self.log("Got an error cloning the test database: %s" % e)
                 sys.exit(2)
+        elif multiprocessing.get_start_method() == "spawn":
+            ondisk_db = sqlite3.connect(target_database_name, uri=True)
+            self.connection.connection.backup(ondisk_db)
```

Comment:
The `if` block has a comment above that states that forking automatically makes a copy of an in-memory database. The `if` block itself relates to databases on disk, and this `elif` block is for in-memory databases but using the `spawn` method.

I think a further comment above the `elif` making it clear that in-memory databases need to be migrated to disk if using `spawn` would be helpful. (Not sure on the wording or details though - not put much thought into it.)

Reply:
I've had a go at rewording this. What do you think?